assets/minecraft/models/item/bow.json

```
@@ -1,34 +1,12 @@
 {
-    "parent": "item/generated",
+    "parent": "item/tools/bow/mia_bow_0",
     "textures": {
         "layer0": "item/bow"
     },
-    "display": {
-        "thirdperson_righthand": {
-            "rotation": [ -80, 260, -40 ],
-            "translation": [ -1, -2, 2.5 ],
-            "scale": [ 0.9, 0.9, 0.9 ]
-        },
-        "thirdperson_lefthand": {
-            "rotation": [ -80, -280, 40 ],
-            "translation": [ -1, -2, 2.5 ],
-            "scale": [ 0.9, 0.9, 0.9 ]
-        },
-        "firstperson_righthand": {
-            "rotation": [ 0, -90, 25 ],
-            "translation": [ 1.13, 3.2, 1.13],
-            "scale": [ 0.68, 0.68, 0.68 ]
-        },
-        "firstperson_lefthand": {
-            "rotation": [ 0, 90, -25 ],
-            "translation": [ 1.13, 3.2, 1.13],
-            "scale": [ 0.68, 0.68, 0.68 ]
-        }
-    },
     "overrides": [
-    { "predicate": {"pulling": 1}, "model": "item/event/camelot/bow/bow_pulling_0"},
-    { "predicate": {"pulling": 1,"pull": 0.65}, "model": "item/event/camelot/bow/bow_pulling_1"},
-    { "predicate": {"pulling": 1,"pull": 0.9}, "model": "item/event/camelot/bow/bow_pulling_2"},
+    { "predicate": {"pulling": 1}, "model": "item/tools/bow/mia_bow_0"},
+    { "predicate": {"pulling": 1,"pull": 0.65}, "model": "item/tools/bow/mia_bow_1"},
+    { "predicate": {"pulling": 1,"pull": 0.9}, "model": "item/tools/bow/mia_bow_2"},
```

`mia_bow_0` on the first line there means that until 0.65 of the animation, the bow will be in the default state. Should be:
```suggestion
    { "predicate": {"pulling": 1"}, "model": "item/tools/bow/mia_bow_1"},
    { "predicate": {"pulling": 1,"pull": 0.9}, "model": "item/tools/bow/mia_bow_2"},
```
 
 if you dont wanna add another position to the arrow.